# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/url/a-element-origin-xhtml.xhtml.ini
# Commit: ef6d81f0b298
# Full Hash: ef6d81f0b2989624a9918ec5d790dc2745db20ef
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-15 09:52:43
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow password with empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 06b12fd41ff1 -r ef6d81f0b298 testing/web-platform/meta/url/a-element-origin-xhtml.xhtml.ini
--- a/testing/web-platform/meta/url/a-element-origin-xhtml.xhtml.ini	Wed Nov 14 19:37:42 2018 +0000
+++ b/testing/web-platform/meta/url/a-element-origin-xhtml.xhtml.ini	Wed Nov 14 19:10:42 2018 +0000
@@ -1,10 +1,4 @@
 [a-element-origin-xhtml.xhtml]
-  [Parsing origin: <https://:@test> against <about:blank>]
-    expected: FAIL
-
-  [Parsing origin: <http://::@c@d:2> against <http://example.org/foo/bar>]
-    expected: FAIL
-
   [Parsing origin: <gopher:/example.com/> against <http://example.org/foo/bar>]
     expected: FAIL
 
@@ -41,15 +35,6 @@
   [Parsing origin: <http::b@www.example.com> against <about:blank>]
     expected: FAIL
 
-  [Parsing origin: <http:/:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
-  [Parsing origin: <http://:b@www.example.com> against <about:blank>]
-    expected: FAIL
-
-  [Parsing origin: <http://:@www.example.com> against <about:blank>]
-    expected: FAIL
-
   [Parsing origin: <i> against <sc:/pa/pa>]
     expected: FAIL
 